minutes/2023/230531Minutes.md

```diff
@@ -0,0 +1,42 @@
+# Minutes May 31st 2023
+
+## Attendees
+- Valentina Hurtado-McCormick
+- Martino Sorbaro
+- Samantha Ahern
+- Toby Hodges
+
+## Agenda items
+
+### Relationship between CAC and LPGC
+The boundary of competences between CAC and LPGC needs to be defined, and this relationship should be publicly clarified.
+We could have another rubric for CAC vs LPGC invovement, or we could add a little bit of material on the CAC vs Maintainer rubric, or we could link to our charter.
+
+We do not entirely understand what is the job of the CAC vs LPGC is. Toby explains that he envisions the CAC having control of lessons and materials, while the LPGC getting involved only at the level of 
+creating new curricula (for example a section for advanced SWC materials, with its own CAC). However, we think a few more detailed aspects might need to be figured out with time.
+We conclude that should meet with the chair or the entire CAC and define together what the boundaries of our respective roles are. Then create a public document detailing them.
+Toby will mention our interest in meeting to the CAC.
+
+Actions: set up a meeting with the CAC chair. Toby will contact them. In that meeting we can determine and draft a rubric.
+
+### Website
+We list a few issues with the SWC website. Among others: older look, which is not properly accessible (due eg. to contrast issues); the blog is outdated but is in the front page,
+giving a feeling of abandonment; some information and sectioning is redundant wrt. the Carpentries website, which leads to a confusing structure.
+
+Martino opened a few issues on the website repo, but was told by Maneesha that there is a plan for a global restructuring so these should be delayed. However, Toby updates us saying that
+the planned restructuring will take time due to some issues that have emerged with an external contractor. Some more urgent and easier aspects might be worth looking at before that happens.
+
+We conclude that we should keep the conversation open asynchronously by categorizing requests that we have in three orders:
+- Small and faster things that can be fixed with simple PRs. Some of us might like/have time to work on these directly in a personal capacity, as long as we coordinate.
+- Urgent and faster things that we might want to discuss with the website team so that they are resolved relatively quickly (before the new website).
+- Things that should wait for the new website, such as the overall template.
+
+Actions: Toby has asked the website maintainers if they are ok with us doing this. Martino has opened an Etherpad on which to coordinate at https://pad.carpentries.org/swc-lpgc-website-review.
+
+### Lesson Program Governor Handbook
+Toby updates us saying that this will be available very soon and will be on docs.carpentries.org.
```

Comment:
Update from Toby during the meeting: the handbook is being incorporated into docs.carpentries.org. This will be related to the content of the charter and to the content of the CAC/LPGC rubric.


Follow-up action by Toby: pull request to add the LPGC Handbook to [docs.carpentries.org](http://docs.carpentries.org/) - https://github.com/carpentries/docs.carpentries.org/pull/911